### ISSUE TITLE
Cannot SSH into gnuroot

### ISSUE THREAD

**Issue by freebrowser1** [corbinlc/GNURootDebian#181]
I installed Gnuroot on an Android 7 cellphone.

It worked fine, but tries to ssh into Gnuroot have no success.
Port 22 is mapped to 2022 by default but ssh'ing from my Mac to ssh -p 2022 root@android resulted in 
Connection closed by <ip> port 2022.

Then I followed the tips from issue 57 (https://github.com/corbinlc/GNURootDebian/issues/57), installing dropbear and set a port 2202 for /etc/default/dropbear and started it.
But the password was not accepted.
Permission denied, please try again.

(I already have 2222 used for DroidSSH, hence the number 2202).

**Comment by ethelfrith** [comment]
I got this same error when installing on a s8
I changed PermitRootLogin from without-password to yes, copied my id_rsa.pub key over then changed it back to without-password
Hope this helps

**Comment by coax75ohm** [comment]
For OpenSSH to work, I found that you need to set
`UsePrivilegeSeparation no`
since it apparently doesn't work inside a proot.

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by satcom886** [comment]
Yes, this issue is still present. I used dropbear on port 22, but each time I SSH into my device I only get "ssh: Could not resolve hostname 192.168.0.150:2022: Name or service not known"

**Comment by AdamMatthewLTM** [comment]
@satcom886 ,

Thank you for your reply. I am recommending this issue remain open as medium priority until further notice. 

Potential duplicates of this post are:

#26 #57 

Thank you all for your patience in these matters,
- Adam

**Comment by corbinlc** [comment]
We are putting our efforts into a new, but similar, app called UserLAnd.  It is now in beta and it should provide ssh and/or vnc working out of the box.  To join the beta of UserLAnd, please go here:  https://play.google.com/apps/testing/tech.ula

**Comment by satcom886** [comment]
@corbinlc Thanks for the update!
